config_defaults_inc.php

```diff
@@ -3618,6 +3618,15 @@
  */
 $g_my_view_boxes_fixed_position = ON;
 
+/**
+ * Toggle whether 'Timeline' boxes show summary of issue after issue ID
+ */
+$g_timeline_show_issue_summary = OFF;
+
+/**
+ * Control the maximum length of each summary shown in timeline when $g_timeline_show_issue_summary is ON.
+ */
+$g_timeline_issue_summary_length = 20;
```

Comment:
I found that $g_public_config_names is for options available for SOAP API. But I am not sure how to update the SOAP API. If someone can help me on this, it will be much appreciated.